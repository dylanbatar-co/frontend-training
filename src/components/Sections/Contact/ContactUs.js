import React from 'react';
import { ContactForm } from '../../ContactForm/ContactForm';

export const ContactUs = () => {
  return (
    <section className="container">
      <h2 className="text-center mb-2">Contact Us</h2>
      <ContactForm />
    </section>
  );
};
