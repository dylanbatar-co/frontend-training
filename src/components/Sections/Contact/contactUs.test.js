import { shallow } from 'enzyme';
import { ContactUs } from './ContactUs';

describe('Contact section Test', () => {
  it('ContactUs component snapshot', () => {
    const component = shallow(<ContactUs />);
    expect(component).toMatchSnapshot();
  });
});
