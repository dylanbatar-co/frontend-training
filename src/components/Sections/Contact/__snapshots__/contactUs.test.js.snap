// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact section Test ContactUs component snapshot 1`] = `
<section
  className="container"
>
  <h2
    className="text-center section-title mb-2"
  >
    Contact Us
  </h2>
  <ContactForm />
</section>
`;
